### ISSUE TITLE
no gene page

### ISSUE THREAD

**Issue by sannagudmundsson** [macarthur-lab/variant-curation-portal#232]
Quite often I get this

![image](https://user-images.githubusercontent.com/44973249/129745063-13b47d84-087a-4734-a326-18338124c480.png)

it does not seem to be gene-specific (and is always for both v3 and v2). I just go to gnomAD and look it up there when it happens but it is pretty frequent so would be great if it is an easy fix. 

Page source if that helps

Line wrap
--
  | <!doctype html>
  | <html lang="en">
  | <head>
  |  
  | <meta charset="utf-8">
  | <meta name="viewport" content="width=device-width, initial-scale=1">
  | <title>Variant curation</title>
  | <link type="text/css" href="/static/bundles/bundle-38459c6ca9024874a50b.css" rel="stylesheet" />
  | </head>
  | <body style="width:100%; height:100%;">
  | <div id="root" style="height:100%; min-height:100%;"></div>
  |  
  | <script type="text/javascript" src="/static/bundles/bundle-38459c6ca9024874a50b.js" ></script>
  | </body>
  | </html>
  |

**Comment by nawatts** [comment]
This looks like an issue with importing data into the curation portal. Not sure how this variant is in the curation portal without any VEP annotations. Maybe it got included because of annotations in RefSeq transcripts (#227) and we later deleted those?

**Comment by sannagudmundsson** [comment]
Yes, that's right. I assume the gene page can't be imported based on the
gene name or something and not the variant? We have quite a lot of these
annoying RefSeq variants.

*---*
*Sanna Gudmundsson, Ph.D., Wallenberg Postdoctoral fellow, O'Donnell-Luria
Lab*
Program in Medical and Population Genetics, Broad Institute of
Massachusetts Institute of Technology and Harvard, Cambridge, MA, USA
Division of Genetics and Genomics, Boston Children's Hospital, Harvard
Medical School, Boston, MA, USA
Analytic and Translational Genetics Unit, Massachusetts General Hospital,
Boston, MA, USA
***@***.***
***@***.***

**Comment by nawatts** [comment]
> I assume the gene page can't be imported based on the gene name or something and not the variant?

Which gene page to show is based on gene IDs in the variant's annotations, and the gnomAD browser currently only supports Ensembl IDs. Looking up the gene page by symbol doesn't work as well since multiple genes may have the same symbol, symbols change over time, etc.

**Comment by nawatts** [comment]
The script in this repository filters transcript consequences based on the provided gene IDs. So as long as Ensembl gene IDs are given, it would not include consequences for RefSeq transcripts.

https://github.com/macarthur-lab/variant-curation-portal/blob/2a466c9437b3eb45e42422fb72e6577c4131bbc8/scripts/get_gnomad_lof_variants.py#L162-L170

Based on Slack messages, my best guess is that variants for this projects were pulled using a different version of the script that filtered by gene **symbol** instead of ID. That would include both Ensembl and RefSeq consequences, since they both have the same gene symbols.
